src/lib/cimb_translator/Config.h

```diff
@@ -89,12 +94,12 @@ namespace cimbar
 			return 2;
 		}
 
-		static constexpr unsigned fountain_chunks_per_frame()
+		static constexpr unsigned fountain_chunks_per_frame(unsigned bitspercell, bool legacy_mode)
 		{
-			return 10;
+			return legacy_mode? 10 : bitspercell << 1;
```

Comment:
fountain_chunks_per_frame needs to be a per-config option. mode B is 12, 4C and 8C are 10.

But this little workaround/hack is a sign that this Config namespace needs a rewrite to be able to better do its job. (this also will reduce the amount of distinct params that get passed into the Decoder/Encoder/etc)